vera/src/palette.rs

```diff
@@ -164,6 +167,54 @@ impl VeraPalette {
 		self.entries.len() * 2
 	}
 
+	/// Derives a palette from the given Gimp gpl file
+	pub fn derive_from_gpl(
+		id: &str,
+		gpl_file: &str,
```

Comment:
For the sake of consistency with other similar functions, it might be better to accept a vec<u8> here similar to the `derive_from_png` function, even if it's a little more fiddly to read a text file as bytes, send it to this function than reinterpret as text. To better support multiple file types (which I think would need to happen as soon as we add a third type), we'd ultimately want this function to be factored out into a trait which accepts data and an enum denoting what kind of input it represents and outputs a palette. So keeping the calls consistent would help toward this.

Reply:
In its current form the `gimp_palette` crate doesn't support parsing the palette from a string or array of bytes.  I was a little torn between using this crate and rolling my own; since the crate hasn't been updated since 2017 the latter is probably a better choice anyway and would allow for the approach you're describing here and below.

Reply:
It's such a simple file format that I think creating a function to read the bytes directly and return a vec of hex tuples right in the `util` crate would be a good way to start, then if you think other people would get use out of it, break it out into its own crate later (at least that's the approach I'd take)